sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -321,6 +324,10 @@ def compute_table_name(row):
 from apache_beam.utils.annotations import deprecated
 from apache_beam.utils.annotations import experimental
 
+import google.cloud.bigquery_storage_v1 as bq_storage
```

Comment:
You probably need to move this under the 'try' statement below. The new dependency is in the gcp extra and not all extras are installed in all environments.

For example this test (https://github.com/apache/beam/runs/3138453711?check_suite_focus=true) is failing in py36 environment and that environment does not install any extras (https://github.com/apache/beam/blob/10e60255683ed14358537ce17ecda54460bebf40/sdks/python/tox.ini#L65). Compare this to "-cloud" version of the same environments.

Similarly you may need to skip tests if the relevant extras are not installed.